tools/styles/tokens/global-vars.css

```diff
@@ -129,6 +129,19 @@ governing permissions and limitations under the License.
     --spectrum-floating-action-button-shadow-color: var(
         --spectrum-floating-action-button-drop-shadow-color
     );
+    --spectrum-table-row-hover-color: var(--spectrum-gray-900);
+    --spectrum-table-row-hover-opacity: 0.07;
+    --spectrum-table-selected-row-background-color: var(
+        --spectrum-informative-background-color-default
+    );
+    --spectrum-table-selected-row-background-opacity: 0.1;
+    --spectrum-table-selected-row-background-color-non-emphasized: var(
+        --spectrum-neutral-background-color-selected-default
+    );
+    --spectrum-table-selected-row-background-opacity-non-emphasized: 0.1;
+    --spectrum-table-row-down-opacity: 0.1;
+    --spectrum-table-selected-row-background-opacity-hover: 0.15;
+    --spectrum-table-selected-row-background-opacity-non-emphasized-hover: 0.15;
     --spectrum-white-rgb: 255, 255, 255;
     --spectrum-white: rgba(var(--spectrum-white-rgb));
     --spectrum-transparent-white-100-rgb: 255, 255, 255;
```

Comment:
can we write this instead?
--spectrum-transparent-white-100-rgb: var(--spectrum-white-rgb)